### PR TITLE
feat: improve cosmos asset page

### PR DESCRIPTION
**Pull request by cjthompson**
## Description

feat: Improve cosmos asset page

* change the send/receive buttons to just be disabled if the wallet doesn't support the chain rather than just not rendered
* remove `StakingOpportunities` from the default asset page

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [X] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [X] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

minimal risk

## Testing

1. enable the REACT_APP_FEATURE_PLUGIN_COSMOS environment variable

#### Asset Details
image

#### Asset Balance
image

#### Dashboard
image

### CODE REVIEW

File: src/components/Delegate/StakingOpportunities.tsx [hunk 1]
```diff
@@ -37,12 +26,10 @@ export const StakingOpportunities = ({ assetId: caip19 }: StakingOpportunitiesPr
...
           {validators.map(validator => (
-            <StakingOpportunitiesRow name={validator.name} />
+            <StakingOpportunitiesRow name={validator.name} key={validator.id} />
           ))}
         </Stack>
       </Card.Body>
     </Card>
   )
-
-  return useRenderForChains([ChainTypes.Cosmos], cosmosStaking)
```
Comment (gomesalexandre) [inline src/components/Delegate/StakingOpportunities.tsx:47]: I like this. Using the `CosmosInvestor` feature flag for conditionally rendering is way cleaner and a better development experience than always having to modify this.

File: src/plugins/cosmos/CosmosAssetAccountDetails.tsx [hunk 2]
```diff
@@ -26,6 +28,7 @@ export const CosmosAssetAccountDetails = ({ assetId: caip19, accountId }: AssetD
           <AssetHeader assetId={caip19} accountId={accountId} />
           {accountId && <AccountAssets assetId={caip19} accountId={accountId} />}
           <AssetAccounts assetId={caip19} accountId={accountId} />
+          {FeatureFlag.CosmosInvestor && <StakingOpportunities />}
```
Comment (gomesalexandre) [inline src/plugins/cosmos/CosmosAssetAccountDetails.tsx:31]: I also like this and agree this is where it should live. If we're going to have plugins, there's no need to "pollute" the generic `<AssetAccountDetails />` with it.

**Review by gomesalexandre — COMMENTED** [review]
That architectural change LGTM. It fits the direction we're taking where most Cosmos stuff should live in plugins, and that's definitely something I'll take into consideration in follow-up PRs moving cosmos-specific modals/views logic to plugins.

File: src/state/slices/portfolioSlice/utils.ts [hunk 3]
```diff
@@ -23,12 +23,14 @@ export type UtxoParamsAndAccountType = {
 // TODO(0xdef1cafe): these should be exported from caip2
 export const ethChainId = 'eip155:1'
 export const btcChainId = 'bip122:000000000019d6689c085ae165831e93'
+export const cosmosChainId = 'cosmos:cosmoshub-4'
```
Comment (theoboldfrazier) [inline src/state/slices/portfolioSlice/utils.ts:26]: Do we want to add osmosis here too? `cosmos:osmosis-1`

**Review by theoboldfrazier — APPROVED** [review]
Left a comment, but overall looks good.

File: src/state/slices/portfolioSlice/utils.ts (hunk 3 above)
Comment (cjthompson) [reply src/state/slices/portfolioSlice/utils.ts:26]: we can but i haven't test it.

**Review by 0xdef1cafe — APPROVED** [review]

File: headers.js [hunk 4]
```diff
@@ -2,9 +2,10 @@ require('dotenv').config()
 
 const cspMeta = Object.entries({
   'default-src': ["'self'"],
-  'child-src': ["'self'",  "blob:", "'report-sample'"],
+  'child-src': ["'self'", 'blob:', "'report-sample'"],
   'connect-src': [
     "'self'",
+    'data:',
```
[the remark this answers is not in the document]
Comment (cjthompson) [reply headers.js:8]: I should pull these changes out into a separate PR except for the UNCHAINED URLs

File: src/components/AssetHeader/AssetHeader.tsx [hunk 5]
```diff
@@ -55,14 +55,13 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
...
       </Flex>
-      {walletSupportsChain ? (
-        <AssetActions
-          isLoaded={isLoaded}
-          assetId={assetId}
-          accountId={accountId ? accountId : singleAccount}
-          cryptoBalance={cryptoBalance}
-        />
-      ) : null}
+      <AssetActions
```
[the remark this answers is not in the document]
Comment (cjthompson) [reply src/components/AssetHeader/AssetHeader.tsx]: I reverted the change

File: src/components/Layout/Header/NavBar/NavBar.tsx [hunk 6]
```diff
@@ -12,19 +12,17 @@ export const NavBar = (props: StackProps) => {
...
-                icon={item.icon}
-                href={item.path}
-                to={item.path}
-                label={translate(item.label)}
-                aria-label={translate(item.label)}
-                data-test={`navbar-${item.label.split('.')[1]}-button`}
-              />
-            </>
+            <MainNavLink
+              key={idx}
```
Comment (cjthompson) [inline src/components/Layout/Header/NavBar/NavBar.tsx:18]: This is to fix an error in the console

**Review by mk1r — APPROVED** [review]
looks good - tested the UI as well

File: src/state/slices/portfolioSlice/utils.ts (hunk 3 above)
Comment (theoboldfrazier) [reply src/state/slices/portfolioSlice/utils.ts:26]: Probably can wait on it for now. We'll need chain-adapters work to include osmosis with supportedChainAdapters anyway.

**Pull request merged**